js/package.json

```diff
@@ -14,6 +14,8 @@
     }
   ],
   "dependencies": {
+    "ajv": "^6.5.2",
```

Comment:
Everything that goes into package.json must use the "~" sign instead of the "^" sign because it locks things down to patch changes rather than more major (possibly breaking) changes.